pontoon/administration/static/js/admin_project.js

```diff
@@ -1,252 +1,284 @@
-$(function() {
-
-  // Before submitting the form
-  $('#admin-form').submit(function () {
-    // Update locales
-    var locales = [{
-      list: 'selected',
-      input: $('#id_locales'),
-    }, {
-      list: 'readonly',
-      input: $('#id_locales_readonly'),
-    }];
-
-    locales.forEach(function(type) {
-      var ids = $('.admin-team-selector .locale.' + type.list).find('li[data-id]').map(function() {
-        return $(this).data('id');
-      }).get();
-      type.input.val(ids);
+$(function () {
+    // Before submitting the form
+    $('#admin-form').submit(function () {
+        // Update locales
+        var locales = [
+            {
+                list: 'selected',
+                input: $('#id_locales'),
+            },
+            {
+                list: 'readonly',
+                input: $('#id_locales_readonly'),
+            },
+        ];
+
+        locales.forEach(function (type) {
+            var ids = $('.admin-team-selector .locale.' + type.list)
+                .find('li[data-id]')
+                .map(function () {
+                    return $(this).data('id');
+                })
+                .get();
+            type.input.val(ids);
+        });
+
+        // Update form action
+        var slug = $('#id_slug').val();
+        if (slug.length > 0) {
+            slug += '/';
+        }
+        $('#admin-form').attr(
+            'action',
+            $('#admin-form').attr('action').split('/projects/')[0] +
+                '/projects/' +
+                slug
+        );
     });
 
-    // Update form action
-    var slug = $('#id_slug').val();
-    if (slug.length > 0) {
-      slug += '/';
-    }
-    $('#admin-form').attr('action', $('#admin-form').attr('action').split('/projects/')[0] + '/projects/' + slug);
-  });
-
-  // Submit form with Enter
-  $('html').unbind("keydown.pontoon").bind("keydown.pontoon", function (e) {
-    if ($('input[type=text]:focus').length > 0 || $('input[type=url]:focus').length > 0) {
-      var key = e.keyCode || e.which;
-      if (key === 13) { // Enter
-        // A short delay to allow digest of autocomplete before submit
-        setTimeout(function() {
-          $('#admin-form').submit();
-        }, 1);
-        return false;
-      }
-    }
-  });
-
-  // Submit form with button
-  $('.save').click(function(e) {
-    e.preventDefault();
-    $('#admin-form').submit();
-  });
-
-  // Manually Sync project
-  $('.sync').click(function(e) {
-    e.preventDefault();
+    // Submit form with Enter
+    $('html')
+        .unbind('keydown.pontoon')
+        .bind('keydown.pontoon', function (e) {
+            if (
+                $('input[type=text]:focus').length > 0 ||
+                $('input[type=url]:focus').length > 0
+            ) {
+                var key = e.keyCode || e.which;
+                if (key === 13) {
+                    // Enter
```

Comment:
I think this comment is off on its own line.

Maybe just remove it? Add a `keyCode == 13` to the comment in line 39 instead, if we want to keep the semantics?